app/routes.py

```
@@ -0,0 +1,40 @@
+from fastapi import APIRouter, HTTPException
+from .models import ComparisonRequest, ComparisonResponse
+from .comparator import OpenAPIComparator
+import os
+
+router = APIRouter()
+comparator = OpenAPIComparator()
+
+@router.post("/compare", response_model=ComparisonResponse)
+async def compare_commits(request: ComparisonRequest):
```

Ruff (0.8.2)

Missing return type annotation for public function `compare_commits`

(ANN201)